__tests__/users/applyFilterPagination.test.js

```diff
@@ -78,8 +78,9 @@ describe('Apply Filter and Pagination Functionality', () => {
   });
 
   it('should update the URL query string when applying filters', async () => {
-    // click on the "Verified" tab
-    await page.click('[data_key="verified"]');
+    // select option between 'in discord' and 'linked accounts' from the dropdoen
```

Comment:
Here also use data test id and remove the comments

Reply:
> Here also use data test id and remove the comments

used data test id, please review